apis/fluent-en_GB/atrium-api-fluent-en_GB/src/jvmMain/kotlin/ch/tutteli/atrium/api/fluent/en_GB/dateSubjectChangers.kt

```diff
@@ -0,0 +1,37 @@
+@file:Suppress(
+    // TODO remove once https://youtrack.jetbrains.com/issue/KT-35343 is fixed
+    "JAVA_MODULE_DOES_NOT_READ_UNNAMED_MODULE"
+)
+
+package ch.tutteli.atrium.api.fluent.en_GB
+
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.logic._logic
+import ch.tutteli.atrium.logic.changeSubject
+import java.time.LocalDate
+import java.time.ZoneId
+import java.util.Date
+
+/**
+ * Turns `Expect<Date>` into `Expect<LocalDate>`.
+ *
+ * The transformation as such is not reflected in reporting.
+ *
+ * @return The newly created [Expect] for the transformed subject.
+ *
```

Comment:
I guess I forgot to write it as task in the issue. can you please create a sample (see for instance: https://github.com/robstoll/atrium/blob/main/apis/fluent-en_GB/atrium-api-fluent-en_GB/src/jvmTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/FileSubjectChangerSamples.kt) and link it here with `@sample`